pmm-app/tests/pages/adminPage.js

```diff
@@ -111,4 +111,20 @@ module.exports = {
       );
     }
   },
+
+  async verifyBackgroundColor(element, expectedColor) {
+    const backgroundColor = await I.grabCssPropertyFrom(element, 'background-color');
+
+    assert.strictEqual(
+      backgroundColor[0],
+      expectedColor,
+      `The Background color of the ${element} is not matching, the actual value is ${backgroundColor}`,
+    );
+  },
+
+  customClearField(field) {
```

Comment:
In some places just a simple `I.clearField()` works great, have you tried it?

Reply:
tried, doesn't work here, will check again.